website/src/html.tsx

```diff
@@ -104,6 +104,13 @@ export default class HTML extends React.Component<HtmlProps> {
                         }}
                     />
 
+                    {/* Add Ceros script */}
+                    <script
+                        type="text/javascript"
+                        src="https://view.ceros.com/scroll-proxy.min.js"
+                        data-ceros-origin-domains="https://view.ceros.com/"
+                    />
+
```

Comment:
I don't think we should load this on every page since not every page is a Ceros page. Instead, we can use the React Helmet package to inject scripts into the head for Ceros-only pages

Reply:
Sure thing. Question: Is there a reason we're not using React's Helmet in the re-platform repo?

Reply:
Next.js has a [similar implementation](https://nextjs.org/docs/api-reference/next/head) out of the box which we use instead